hypothesis-python/src/hypothesis/core.py

```diff
@@ -529,7 +530,7 @@ def test(*args, **kwargs):
                 result = self.test(*args, **kwargs)
                 finish = benchmark_time()
                 internal_draw_time = sum(data.draw_times[initial_draws:])
-                runtime = (finish - start - internal_draw_time) * 1000
+                runtime = datetime.timedelta(microseconds=((finish - start - internal_draw_time) * 1000000))
```

Comment:
If we pass `seconds=` instead of microseconds, we don't need the constant factor :smile: